UnitsNet/CustomCode/Quantities/TemperatureDelta.extra.cs

```diff
@@ -34,5 +34,11 @@ public partial struct TemperatureDelta
         {
             return Energy.FromJoules(entropy.JoulesPerKelvin * temperatureDelta.Kelvins);
         }
+
+        /// <summary>Get a scalar from a <see cref="TemperatureDelta"/> multiplied by a <see cref="CoefficientOfThermalExpansion"/>.</summary>
+        public static double operator *(TemperatureDelta temperatureDelta, CoefficientOfThermalExpansion cte)
+        {
+            return cte * temperatureDelta;
```

Comment:
Can you make this an explicit computation with units as you did for CTE?
It reduces indirection and the potential for infinite recursion.

Reply:
Yes, no problem. Change made.